docs/how-to-guides/renkulab/ssh-into-sessions.rst

```diff
@@ -0,0 +1,255 @@
+.. _ssh_into_sessions:
+
+SSH into RenkuLab Sessions
```

Comment:
> * I have a feeling the VSCode extensions will be an issue - should we point the user to the [VSCode doc that tells you how to add extensions by default?](https://code.visualstudio.com/docs/remote/ssh#_always-installed-extensions)
I'm not sure what you mean here - are you referring to _other_ VSCode extensions that the user will have to enable for SSH sessions?

> * I believe the first time you connect with VSCode, the file browser is in the `jovyan` home directory and _not_ the project directory - this might be confusing and could be mentioned in the docs?
This instruction is in there (Step 3 in the VSCode section)- do you have a suggestion for how I could make it clearer? 

> * I'm wondering if "SSH into a session" and "Use a local IDE with Renkulab sessions" should be two separate how-to guides. If I'm a VSCode user I might not immediately know that "SSH into sessions" will translate into "I can use local VSCode with Renku sessions". We could split off the VSCode section and point to the SSH section as a "prerequisite"?
That's a good point. For now, I included both in the doc's Title